packages/components/src/number-control/types.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import type { HTMLInputTypeAttribute, InputHTMLAttributes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import type { InputControlProps } from '../input-control/types';
+
+export type NumberControlProps = Omit<
+	InputControlProps,
+	'isDragEnabled' | 'min' | 'max' | 'required' | 'type' | 'value'
```

Comment:
Good question! Some of them, because we need a different type (`min`, `max`, `value`). Almost all of them, because we need a different `@default` value. I haven't yet found a way to "import" the JSDoc description only, or override the `@default` only 😕

Reply:
Yeah, that's unfortunate.

Would it make sense to still re-use the type for `InputControlProps` for those props? E.g.

`isDragEnabled?: InputControlProps[ 'isDragEnabled' ];`

Reply:
Yeah, I think that might be better 👍